cdap-common/bin/functions.sh

```diff
@@ -207,13 +207,7 @@ cdap_stop_pidfile() {
 #
 cdap_check_pidfile() {
   local readonly __pidfile=${1} __label=${2:-Process}
-  if [[ -f ${__pidfile} ]]; then
-    local readonly __pid=$(<${__pidfile})
-    if [[ $(cdap_status_pidfile ${__pidfile} ${__label} >/dev/null) ]]; then
-      echo "${__label} running as PID ${__pid}. Stop it first."
```

Comment:
So suppress the `cdap_status_pidfile` output like was done here and give our own if it's running? I can see the confusion a little, but if I try a start while it's already running, the end result gives me what I wanted, a started CDAP Standalone. I'm kinda up in the air about this.